src/platform_impl/macos/window.rs

```diff
@@ -804,7 +804,9 @@ impl WinitWindow {
         if let Some(ref fullscreen) = fullscreen {
             let new_screen = match fullscreen {
                 Fullscreen::Borderless(Some(monitor)) => monitor.clone(),
-                Fullscreen::Borderless(None) => self.current_monitor_inner(),
+                Fullscreen::Borderless(None) => {
+                    self.current_monitor_inner().expect("expected screen")
```

Comment:
Wait, why do we crash here? Could we just return and no-op?

Reply:
`set_fullscreen` has no return value, the user always assumes it will succeed. I'm not sure if no-op here wil cause a state error.

Reply:
It won't compositor/system could ignore full-screen request. That's sort of expected. You also don't really know whether it succeeded or not until you query.

Reply:
Ok